SimTracker/TrackAssociation/plugins/TrackMCQuality.cc

```diff
@@ -39,7 +39,7 @@
 // class decleration
 //
 
-class TrackMCQuality : public edm::global::EDProducer<> {
```

Comment:
this change isn't described in the PR description.
It has nothing to do with the track algo changes.
[what was wrong with just making a separate PR for the TrackMCQuality?]

Please describe in the PR description and the subject of the PR as well.